src/applications/apps.js

```diff
@@ -25,6 +24,50 @@ import { assign } from "../utils/assign";
 
 const apps = [];
 
+export function getAppChanges() {
+  const appsToUnload = [],
+    appsToUnmount = [],
+    appsToLoad = [],
+    appsToMount = [];
+
+  // We re-attempt to download applications in LOAD_ERROR after a timeout of 200 milliseconds
+  const currentTime = new Date().getTime();
+
+  apps.forEach((app) => {
+    const appShouldBeActive =
+      app.status !== SKIP_BECAUSE_BROKEN && shouldBeActive(app);
+
+    switch (app.status) {
+      case LOAD_ERROR:
+        if (currentTime - app.loadErrorTime >= 200) {
```

Comment:
is this new functionality? I'm not seeing the same behavior in the code you removed.

(I like it, btw, just wanted to make sure)

Reply:
oh actually, we should only push into `appsToLoad` if it should be active, right?

Reply:
This feature existed before this PR, see https://github.com/single-spa/single-spa/pull/546/files#diff-b954ca9fc4221dce0e4e9cffc6a642c9L59. I believe @blittle added it a while back

Reply:
I remember talking about it, but I must have missed it in the code you deleted. lemme double check again

Reply:
Here's the existing test for it https://github.com/single-spa/single-spa/blob/38ada88af5cc0188b80281bc6f275e180bf54ac4/spec/apps/invalid-load-function/invalid-load-function.spec.js#L124